net-core/Ical.Net/Ical.Net/DataTypes/PeriodList.cs

```diff
@@ -30,7 +31,7 @@ public PeriodList(string value) : this()
             CopyFrom(serializer.Deserialize(new StringReader(value)) as ICopyable);
         }
 
-        protected bool Equals(PeriodList other)
+        public bool Equals(PeriodList other)
         {
             return string.Equals(TzId, other.TzId) && CollectionHelpers.Equals(Periods, other.Periods);
```

Comment:
While you're here, let's make all `string.Equals` use `StringComparer.OrdinalIgnoreCase`.